FuncGodot Manual/pages/tips_runtime_building.html

```diff
@@ -0,0 +1,112 @@
+        func_godot_map.verify_and_build()</code></pre>
+    <blockquote><i>NOTE: You can extend this to have more commands like God mode or Debug mode as your game needs, and use them when launching in editor.<br></i></blockquote>
+    <h2>Setting Up Your Editor</h2>
+    <p>In every map editor you'll need to specify both an <b>executable</b> and <b>parameters</b> in your launch or build options. The executable can be either the <b>exported binary of your game</b> or the <b>Godot Editor binary</b> if launching from a Godot project. To run from your game from the Godot project, be sure to use the <code>--path</code> parameter, as defined in <a href="https://docs.godotengine.org/en/stable/tutorials/editor/command_line_tutorial.html" target="_blank"><b>Godot's Command Line Interface options</b></a>.</p>
+    <h3>TrenchBroom</h3>
+    <p><a href="https://trenchbroom.github.io/manual/latest/#launching_game_engines"><b>The TrenchBroom Reference Manual</b></a> covers launch option configurations quite thoroughly. If you chose to use the Godot Editor for your launch executable, make sure to start your parameters with the <code>--path</code> command. You'll then add the <code>--run_map::</code> build command we defined earlier, using the built-in TrenchBroom variable <code>${MAP_BASE_NAME}</code> to get the name of the currently opened map file.</a>.</p>
+    <p>An example of set of parameters to launch from the Godot Editor will look something like this: <code>--path "C:\GAME DEV\FuncGodot\test_project" -- run_map::${MAP_BASE_NAME}.map</code></p>
+    <p>
+        To run your map from TrenchBroom, navigate to <code>Run > Launch Engine</code>.
+        Go to <code>Configure engines...</code>, add a game engine profile, and provide a name and path to the executable for this. For this example, we use the path to the Godot editor executable.
+        Once you're happy with your engine configuration, click close. You will now see this as an option you can select. Select it.
+        <p align="center"><img src="../images/tips_editorbuild_trenchbroom1.png"><br></p>
+        <p>Now when you press Launch your game should launch using the parameters you defined!</p>
+    </p>
+
+    <h3>NetRadiant Custom</h3>
+
+    <p>NetRadiant Custom utilizes a <code>default_build_menu.xml</code> file, located in your game's gamepack folder for its build processes. This file is automatically generated by NetRadiant Custom after setting up your gamepack configuration, though it needs some additional setup in order to work with your project.</p>
+
+    <p>To do so, you must define at least one or more build <b>commands</b>, which may refer to build <b>variables</b> also defined here, or one of the default variables the editor provides for you. To preview these defaults, or any of your defined commands and their current values, select <code>Build > Customize...</code> in the top menu bar and check the <code>build variables</code> option.</p>
```

Comment:
Add text to reflect [func_godot's ability to now include these for a real source the user can *go* to to type them in instead of assuming things:

```
<p>To do so, you must define at least one or more build <b>commands</b>, which may refer to build <b>variables</b> also defined here, or one of the default variables NetRadiant provides for you.  You may either write them directly in the file yourself, or define them in the NetRadiantCustom Game Config resource. To preview these defaults, or any of your defined commands and their current values, select <code>Build > Customize...</code> in the top menu bar and check the <code>build variables</code> option.</p>```

Reply:
Ackshuelly it's called the NetRadiant Custom GamePack Config resource.

Reply:
Resolved in latest commit